app/views/result_filters/location/new.html.erb

```diff
@@ -50,12 +50,14 @@
                 <%=
                   form.text_field(
                     :lq,
+                    id: 'location',
                     value: "#{request.params[:lq]}",
                     class: "govuk-input",
-                    data: {qa: "location-query"},
+                    data: {qa: "location-query", url: "/location-suggestions"},
```

Comment:
Is this url same as location_suggestions_path (railsy one)

Reply:
Yep. You rails ninja